internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,426 @@
+	for _, topics := range details.topics {
+		if topics.TopicName == topicName {
+			// Topic already exists
+			log.CliLogger.Info("Topic is already present.")
+			if !overwrite {
+				// Do not overwrite existing topic. Move to the next topic.
+				return true, nil
+			}
+			// Overwrite topic configs
+			log.CliLogger.Info("Overwriting topic configs")
+			_, err := kafkaRest.CloudClient.UpdateKafkaTopicConfigBatch(details.clusterId, topicName, kafkarestv3.AlterConfigBatchRequestData{Data: updateConfigs})
+			if err != nil {
+				return true, fmt.Errorf("unable to update topic configs: %v", err)
+			}
+			if description != "" {
+				if err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName,
+					description); err != nil {
+					return true, fmt.Errorf("unable to update topic description: %v", err)
+				}
+			}
+			return true, nil
+		}
+	}
+	log.CliLogger.Infof("Topic not found. Adding a new topic: %s", topicName)
+	if _, httpResp, err := kafkaRest.CloudClient.CreateKafkaTopic(details.clusterId,
+		topicRequestData); err != nil {
+		restErr, parseErr := kafkarest.ParseOpenAPIErrorCloud(err)
+		if parseErr == nil && restErr.Code == ccloudv2.BadRequestErrorCode {
+			// Print partition limit error w/ suggestion
+			if strings.Contains(restErr.Message, "partitions will exceed") {
+				return false, errors.NewErrorWithSuggestions(restErr.Message, errors.ExceedPartitionLimitSuggestions)
+			}
+		}
+		return false, kafkarest.NewError(kafkaRest.CloudClient.GetUrl(), err, httpResp)
+	}
+	log.CliLogger.Infof("Added topic %s to cluster %s.\n", topicName, details.clusterId)
+
+	if description != "" {
+		if err = addTopicDescription(details.srClient, details.srContext, details.clusterId+":"+topicName,
+			description); err != nil {
+			return false, fmt.Errorf("unable to update topic description: %v", err)
+		}
+	}
+	return false, nil
+}
+
+func addTopicDescription(srClient *srsdk.APIClient, srContext context.Context, qualifiedName string, description string) error {
+	atlasEntity := srsdk.AtlasEntityWithExtInfo{
+		ReferredEntities: nil,
+		Entity: srsdk.AtlasEntity{
+			Attributes: map[string]any{
+				"description":   description,
+				"qualifiedName": qualifiedName,
+			},
+			TypeName: "kafka_topic",
+		},
+	}
+	_, err := srClient.DefaultApi.PartialUpdateByUniqueAttributes(srContext,
+		&srsdk.PartialUpdateByUniqueAttributesOpts{AtlasEntityWithExtInfo: optional.NewInterface(atlasEntity)})
+	return err
+}
+
+func resolveSchemaType(contentType string) string {
+	if strings.Contains(contentType, "avro") {
+		return "AVRO"
+	} else if strings.Contains(contentType, "json") {
+		return "JSON"
+	} else {
+		return "PROTOBUF"
+	}
+}
+
+func registerSchema(details *accountDetails, topicName string, components Components) (int32, error) {
+	// Registering the schema
+	// Subject name follows the TopicNameStrategy
+	subject := topicName + "-value"
+	output.Printf("Registering schema under the subject %s\n", subject)
```

Comment:
Can we move this message to after it's been created, and put this message in past tense?